### ISSUE TITLE
The connection attempt failed because the connected party did not respond correctly after a period of time or the connected host did not respond.

### ISSUE THREAD

**Issue by hongyonghan** [MaheshSuranga/Overseer-Engine#1]
I ran your program and encountered this problem. File "F:\pythonProject\Overseer-Engine\utils\videoSender.py", line 9, in send_train_video
     soc.connect((SERVER_IP,8080))
TimeoutError: [WinError 10060] The connection attempt failed because the connected party did not respond correctly after a period of time or the connected host did not respond.
Can you solve it for me? Thank you

**Comment by MaheshSuranga** [comment]
@hongyonghan yep. That's because here we use another separate server machine to process the `Gait Recognition` part and it has a separate application(Because it is not possible to run both apps in a single machine.). After capturing video frames this app sends the frames to that machine which is identified by SERVER_IP and 8080 port no. Then that machine will do the Gait Recognition part and send the result back to the main machine which is this.

That gait recognition app is mainly developed by another colleague and he did not push his code to GitHub. But don't worry I will inform him immediately to push his program and let you know the instructions to set up the environment. Extremely sorry for your inconvenience.

**Comment by MaheshSuranga** [comment]
> I ran your program and encountered this problem. File "F:\pythonProject\Overseer-Engine\utils\videoSender.py", line 9, in send_train_video
> soc.connect((SERVER_IP,8080))
> TimeoutError: [WinError 10060] The connection attempt failed because the connected party did not respond correctly after a period of time or the connected host did not respond.
> Can you solve it for me? Thank you

@hongyonghan yep. That's because here we use another separate server machine to process the Gait Recognition part and it has a separate application(Because it is not possible to run both apps in a single machine.). After capturing video frames this app sends the frames to that machine which is identified by SERVER_IP and 8080 port no. Then that machine will do the Gait Recognition part and send the result back to the main machine which is this.

That gait recognition app is mainly developed by another colleague and he did not push his code to GitHub. But don't worry I will inform him immediately to push his program and let you know the instructions to set up the environment. Extremely sorry for your inconvenience.

**Comment by hongyonghan** [comment]
Thank you very much for your reply. I am also doing related research recently and want to know your method of real-time gait recognition.

**Comment by MaheshSuranga** [comment]
> Thank you very much for your reply. I am also doing related research recently and want to know your method of real-time gait recognition.

Thanks for your interest in our work. I will post our report here for your reference to get an idea about what is our way till the other project is push to GitHub.
[FYP - Final Report.pdf](https://github.com/MaheshSuranga/Overseer-Engine/files/6290016/FYP.-.Final.Report.pdf)

**Comment by hongyonghan** [comment]
Thank you for your help. If possible, I would like to understand the relevant code of the project to reproduce the entire project
